portal.go

```diff
@@ -450,6 +452,14 @@ func (portal *Portal) markRead(intent *appservice.IntentAPI, eventID id.EventID,
 	if intent == nil {
 		return nil
 	}
+
+	// Only send the read update if it is more recent than the last read time
+	if portal.ReadAt.After(readAt) {
+		return nil
+	} else {
+		portal.ReadAt = readAt
+	}
+
```

Comment:
This seems unnecessary and technically incorrect, it won't do anything after a restart anyway since ReadAt isn't saved

Reply:
I wasn't sure what would happen to matrix if it receives a read receipt for an older message than the most recent one than it already has. I understand the property isn't saved on restart, but that isn't the point either. The idea is to make sure that if BlueBubbles sends an `updated-message` event and the code believes it is a read receipt but turns out to be an older message than the most recent read message, this code will block it from being sent.

Reply:
It doesn't account for who read the message, so it doesn't work correctly. Matrix read receipts can't move backwards, so the code can just be removed